### DISCUSSION TITLE
Suggestion: Show task list progress for issues on project board

### DISCUSSION THREAD

**Discussion by DorkMatter** [community/community#11896] [Projects and Issues] [answered]
It would be nice to be able to display the task list progress for issues on the project board, that helps create a quick overview of how close a task is to moving from In Development to Done.

**Comment by jason-dour** [comment]
Agreed. Basic issue/task tracking from the Project view would fill a huge gap.

**Comment by Patrick-van-Halm** [comment]
I agree. This can use the markdown checkboxes to keep track of the actual progress. Indentation makes sub progress tracking.

**Comment by andrewhsu** [comment, marked as answer]
is this suggestion a duplicate of https://github.com/github/feedback/discussions/7575 ?

**Reply by Patrick-van-Halm** [reply]
When it comes to my comment yes I apparently missed that issue. I don't know if the original author means the duplicate.

**Reply by DorkMatter** [reply]
Duplicate of specifically [this](https://github.com/github/feedback/discussions/7575#discussioncomment-1693706), yes, hadn't seen that issue either, thanks for pointing it out!
